Tests/AsyncAlgorithmsTests/TestMerge.swift

```diff
@@ -13,117 +13,115 @@
 import AsyncAlgorithms
 
 final class TestMerge2: XCTestCase {
-  func test_even_values() async {
-    let merged = merge([1, 2, 3].async, [4, 5, 6].async)
+  private func assertSequencesAreMerged(first: [Int], second: [Int]) async {
```

Comment:
You make a good point. Du you suggest to avoid code factorisation here ?

Reply:
I think test failures are showing the correct location if we forward `#file` and `#line` arguments, but maybe I'm mistaken.

Reply:
yea, repetition is actually really good for tests; granted it may be really verbose - it makes each test to be self contained and distinctly readable for the parts that are being test.

Reply:
@phausler as suggested I've rolled back the factorisation.